BFH_USZ_PICC/BFH_USZ_PICC.UWP/DependencyServices/ReminderNotification_UWP.cs

```diff
@@ -18,33 +18,26 @@ public class ReminderNotification_UWP : IReminderNotification
         //List with all planned notifications
         private List<ScheduledToastNotification> plannedNotifications = new List<ScheduledToastNotification>();
 
-        void IReminderNotification.AddNotification(DateTimeOffset maintenanceReminderStartDateTime, int dailyInterval, int maintenanceReminderRepetition, string title, string body)
+        void IReminderNotification.AddNotification(DateTimeOffset maintenanceReminderStartDateTime, int dailyInterval, int maintenanceReminderRepetition, string title, string body, bool isUnlimited)
         {
             ToastNotifier = ToastNotificationManager.CreateToastNotifier();
 
             XmlDocument toastXml = createToastXML(title, body);
-            //local variable to check for how many weeks the user wants a notifications 
-            int reminderRepetition = 0;
 
-            //this loop checks how many reminder repetation the user wants to plan
-            while (reminderRepetition <= maintenanceReminderRepetition)
+            // Checks if the user wants a to set a repetition limit or if he wants an unlimited reminder. In this case, 200 notifications will be generated.
+            if (isUnlimited)
             {
-                DateTimeOffset notificationDateTime = maintenanceReminderStartDateTime.AddDays(reminderRepetition * dailyInterval);
-
-                //Checks if the notification date time is in the future. If not, the notification can not be planned.
-                if (notificationDateTime > DateTimeOffset.Now)
-                {
-                    var stn = new ScheduledToastNotification(toastXml, notificationDateTime);
-                    plannedNotifications.Add(stn);
-                }
+                int REPETITIONFOR200Times = 200;
```

Comment:
Naming. Either it is a const, then it is declared at the top of the class and ALL CAPS or it is not, then it is written in lower camel case.